axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/AxonServerQueryBus.java

```diff
@@ -941,6 +982,7 @@ public void handle(QueryRequest query, ReplyChannel<QueryResponse> responseHandl
 
         @Override
         public FlowControl stream(QueryRequest query, ReplyChannel<QueryResponse> responseHandler) {
+            logger.info("stream query " + Thread.currentThread());
```

Comment:
A similar log-statement remark as Allard made: If we need the statement at all, it would be on `debug` level.